### DISCUSSION TITLE
[Feature Request] Tabs增加扩展插槽

### DISCUSSION THREAD

**Discussion by carlosyan1807** [element-plus/element-plus#1011] [Ideas]
参考 Ant Design

![image](https://user-images.githubusercontent.com/50797685/100970731-732efa80-3570-11eb-8578-c545595ca924.png)

期望
```html
<el-tabs>
  <el-tab-pane name="1"> xxx </el-tab-pane>
  <el-tab-pane name="2"> xxx </el-tab-pane>
  <template #extraContent>
    <i class="el-icon-delete" />
    <el-button type="primary">xxx</el-button>
    <!-- more -->
  </template>
</el-tabs>
```

比较常见的需求，暂时是用css实现了
但当tabs过多出现横向滚动时，会出现el-tabs__nav-prev和el-tabs__nav-next，导致错位
希望增加插槽，且插槽应该在滚动区外，也就是next后面

曾发Issues #800 见启用了新的Discussions 重发一次

**Comment by zazzaz** [comment]
nice to have it

**Comment by likeswinds** [comment]
需要此功能，用来动态控制标签页添加删除，最好不过了

**Comment by nikkufans** [comment]
like this: https://github.com/element-plus/element-plus/pull/14445

**Comment by Cjsah** [comment]
刚要发Discussions就看到了这个
